folly/experimental/symbolizer/Dwarf.cpp

```diff
@@ -19,7 +19,11 @@
 
 #include <type_traits>
 
-#include <dwarf.h>
+#if FOLLY_HAVE_LIBDWARF_DWARF_H
+# include <libdwarf/dwarf.h>
```

Comment:
Alternative implementation that just appends to CPPFLAGS: tedjp/libdwarf-moved-cppflags-fb